### DISCUSSION TITLE
Malzeme stok bilgisi

### DISCUSSION THREAD

**Discussion by sedayilmazba** [acikkaynak/deprem-yardim-projesi#45] [Ideas]
Hangi bölgeye ne malzeme gerekli , ne kadar gerekli ? Aciliyatına göre kademelendirilebilir öncelik sırasına göre yardım gönderebileceklere bilgilendirme yapılabilir .

**Comment by s3audi** [comment]
Stok yönetimi çok çok önemli size katılıyorum. Bu konu belki dışarıdan yönetilen ERP ile entegre olacak şekilde sisteme dahil edilebilir. Bu ERP AFAD'ın kullandığı ve sadece gerekenlerin paylaşıldığı bir arayüze sahip olabilir. Deprem.io en önemli linklerin kesişim noktası olabilir.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebinizi hali hazırda [deprem-io](https://github.com/acikkaynak/deprem-io) içerisinde sürdürülmektedir.
